src/components/NcButton/NcButton.vue

```diff
@@ -35,46 +35,46 @@ It can be used with one or multiple actions.
 		<p>Primary</p>
 		<NcButton
 			aria-label="Example text"
-			:disabled="disabled"
-			:size="size"
+			:disabled
+			:size
+			:text
 			variant="tertiary-no-background">
 			<template v-if="style.indexOf('icon') !== -1" #icon>
 				<Video
 					:size="20" />
 			</template>
-			<template v-if="style.indexOf('text') !== -1">Example text</template>
 		</NcButton>
 		<NcButton
 			aria-label="Example text"
-			:disabled="disabled"
-			:size="size"
+			:disabled
+			:size
+			:text
```

Comment:
TIL, that you can shorthand those. Is this only available in Vue 3?

Reply:
yes thats Vue3 only. Its basically the same as native HTML where you can set e.g. `<input disabled>`.

Reply:
More specifically, Vue 3.5
But it isn't like `<input disabled>` (boolean prop shortcut for `<input :disabled="true">` from Vue 2, but a JS like shortcut, like `{ disabled: disabled }` for `{ disabled }`